### PR TITLE
Update breadcrumb styling

### PR DESCRIPTION
**Pull request by systimotic**
I updated the breadcrumb styling.

Before:
20170926_09-32-48_firefox
After:
20170926_09-29-49_firefox
After when hovering:
20170926_09-31-33_firefox

I removed a lot of the spacing around it and added the background back in, in order to both make the content more prominent and separate the breadcrumb.

### CODE REVIEW

**Comment by Bouncey** [comment]
@systimotic rebase please 👍

File: src/templates/Article.jsx [hunk 1]
```diff
@@ -33,18 +31,10 @@ function Article(props) {
...
-        </Col>
-      </Row>
-      <Row>
-        <Col xs={ 12 }>
-          <div
-            dangerouslySetInnerHTML={{ __html: html }}
-          />
-        </Col>
-      </Row>
+      <Breadcrumbs path={ pathname } />
```
Comment (Bouncey) [inline src/templates/Article.jsx]: OK 😢

**Comment by systimotic** [comment]
@Bouncey Rebased!

**Comment by Bouncey** [comment]
:sparkles: :sparkles: :sparkles: :sparkles:

**Pull request merged**